client/database/models.go

```diff
@@ -0,0 +1,384 @@
+	"github.com/jackc/pgx/v5"
+	_ "github.com/mattn/go-sqlite3" // Импортируем драйвер SQLite
+)
+
+// InitDB инициализирует базу данных и создает таблицы
+func InitDB() (*sql.DB, error) {
+	db, err := sql.Open("sqlite3", "./database/data.db")
+	if err != nil {
+		return nil, err
+	}
+
+	// Применение миграций
+	driver, err := sqlite3.WithInstance(db, &sqlite3.Config{})
+	if err != nil {
+		return nil, err
+	}
+
+	m, err := migrate.NewWithDatabaseInstance(
+		"file://database/migrations",
+		"sqlite3://database/migrations/data.db", driver)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := m.Up(); err != nil && err != migrate.ErrNoChange {
+		return nil, err
+	}
+
+	return db, nil
+}
+
+// AddSecret добавляет новую запись в базу данных
+func AddSecret(db *sql.DB, s *proto.CreateSecretRequest) error {
+	insertSQL :=
+		`INSERT INTO secrets (name, type, content, ownerID, keyID)
+		VALUES (?, ?, ?, ?, ?)`
+	_, err := db.Exec(insertSQL, s.Name, s.Type, s.Content, s.UserId, s.KeyId)
+	return err
+}
+
+// DeleteSecret удаляет запись по ID
+func DeleteSecret(db *sql.DB, s *proto.DeleteSecretRequest) error {
+	deleteSQL := `
+		DELETE FROM secrets
+		WHERE name = ? AND secrets.ownerID = ?;
+		`
+	_, err := db.Exec(deleteSQL, s.Name, s.UserId)
+	return err
+}
+
+func LoginUser(db *sql.DB, userLogin string, userPassword string) (int, error) {
+	var userID int
+	if userLogin == "" || userPassword == "" {
+		err := errors.New("login or password is empty")
+		if err != nil {
+			logger.Warnf("LoginUser: " + err.Error())
+			return -1, err
+		}
+	}
+	selectUserWithPass := `SELECT userID FROM users WHERE userLogin = ? AND userPassword = ?`
+
+	hash := md5.Sum([]byte(userPassword))
+	hashedPass := hex.EncodeToString(hash[:])
+
+	result := db.QueryRow(selectUserWithPass, userLogin, hashedPass)
+	switch err := result.Scan(&userID); err {
+	case pgx.ErrNoRows:
+		return -1, nil
+	case nil:
+		return userID, nil
+	case err:
+		logger.Warnf("Query LoginUser: " + err.Error())
+		return -1, nil
+	}
+	return userID, nil
+}
+
+func GetSecretsList(db *sql.DB, userID int) ([]*proto.SecretsList, error) {
+	var secrets []*proto.SecretsList
+
+	if userID <= 0 {
+		err := errors.New("wrong userID")
+		if err != nil {
+			logger.Warnf("LoginUser: " + err.Error())
+			return secrets, err
+		}
+	}
+
+	rows, err := db.Query("SELECT name, type FROM Secrets WHERE ownerID = ?", userID)
+	if err != nil {
+		return secrets, err
+	}
+	defer rows.Close()
+
+	for rows.Next() {
+		var secret proto.SecretsList
+		if err := rows.Scan(&secret.Name, &secret.Type); err != nil {
+			log.Fatal(err)
+		}
+		secrets = append(secrets, &secret)
+	}
+
+	if err := rows.Err(); err != nil {
+		return secrets, err
+	}
+
+	return secrets, nil
+}
+
+func GetSecret(db *sql.DB, name string, userID int) (*proto.GetSecretResponse, error) {
+	var secret proto.GetSecretResponse
+
+	if userID <= 0 {
+		err := errors.New("wrong userID")
+		if err != nil {
+			logger.Warnf("GetSecret: " + err.Error())
```

Comment:
```suggestion
			logger.Errorf("GetSecret: " + err.Error())
```
Это касается всех ошибок с последующим return. Уровень Warn ставится тогда, когда мы собираемся ошибку проигнорировать.